cmd/podman/tag.go

```diff
@@ -12,7 +12,7 @@ var (
 
 	tagDescription = "Adds one or more additional names to locally-stored image."
 	_tagCommand    = &cobra.Command{
-		Use:   "tag [flags] IMAGE TAG [TAG...]",
+		Use:   "tag [flags] IMAGE NAME [NAME...]",
```

Comment:
Should this be TARGET-NAME?  TO be consistent with the man page, or should the man page just reference NAME?

Reply:
Okay, I changed it to `TARGET_NAME` to keep it consistent.